include/ignition/gazebo/ServerConfig.hh

```diff
@@ -36,6 +37,22 @@ namespace ignition
     // Forward declarations.
     class ServerConfigPrivate;
 
+    /// \brief Type of SDF source.
+    enum class SourceType
```

Comment:
I think this should either be inside `ServerConfig` or have a more descriptive name so we don't accidentally clash inside the `ignition::gazebo` namespace in the future.

Reply:
Moved to be inside `ServerConfig` in [7d33cf0](https://github.com/ignitionrobotics/ign-gazebo/pull/1333/commits/7d33cf0aafacefa3a4bdaa1522ce02b7d7b5263b)